source/manual/alerts/onsite-backups.html.md

```diff
@@ -4,21 +4,21 @@ title: Onsite backups failed
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2018-03-26
+last_reviewed_on: 2018-06-04
 review_in: 6 months
 ---
 
-The backup machine (e.g. `backup-1.management.integration`) collects
-backups from the various data stores.
+The backup machine (e.g. `backup-1.management.integration`) collects backups from the various data stores at 9am every morning.
 
-The location of the backups is defined in
-[govuk-puppet](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_backup.pp)
+The location of the backups is defined in [govuk-puppet](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_backup.pp)
```

Comment:
Minor: this is missing a period at the end.

Reply:
Amended!